tensorflow/python/estimator/inputs/queues/feeding_functions.py

```diff
@@ -336,6 +395,10 @@ def _enqueue_data(data,
           "data must be either a numpy array or pandas DataFrame if pandas is "
           "installed; got {}".format(type(data).__name__))
 
+    if shuffle and pad_data:
+      raise NotImplementedError(
+        "for now, there is now way to pad and shuffle data at the same time")
```

Comment:
Please change the error message to "Padding and shuffling at the same time is not implemented."